src/component/courseList.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { CardContent, Typography, Card } from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
+
+const styles = {
+  card: {
+    maxWidth: '100%'
+  },
+  title: {
+    fontSize: '1rem',
+    fontWeight: 700
+  }
+}
+
+const CourseList = props => {
+  const { classes } = props
+  const courses = props.courses.map((list) => (
+    <Card className={classes.card} key={list.id}>
```

Comment:
The cards would look better if there was a bottom margin for each of them, and a margin between the map and the first card.